client/crash_report_database_generic.cc

```diff
@@ -321,6 +326,22 @@ base::FilePath CrashReportDatabaseGeneric::DatabasePath() {
   return base_dir_;
 }
 
+void CrashReportDatabaseGeneric::LaunchCrashReporter(
+    const base::FilePath& crash_reporter,
+    const base::FilePath& crash_envelope) {
+#if BUILDFLAG(IS_LINUX)
+  SpawnSubprocess(
+      {
+          crash_reporter.value(),
+          crash_envelope.value(),
+      },
+      nullptr,
+      -1,
+      !crash_reporter.IsAbsolute(),
+      nullptr);
+#endif
+}
```

Comment:
As this only works on Linux, should id be moved to the linux-specific file?

Reply:
Not sure why it's called "generic", but it's only compiled for Linux and Android:
https://github.com/getsentry/crashpad/blob/3f26be16a95d61eefbee25af8355debc6ec36707/client/CMakeLists.txt#L47-L56